src/Sylius/Bundle/ChannelBundle/Behat/ChannelContext.php

```diff
@@ -170,15 +170,15 @@ private function configureChannel(ChannelInterface $channel, $localeCodes = null
         if ($shippingMethodNames) {
             $shippingMethodNames = array_map('trim', explode(',', $shippingMethodNames));
             foreach ($shippingMethodNames as $shippingMethodName) {
-                $shippingMethod = $shippingMethods = $this->getRepository('shipping_method')->findOneBy(['name' => $shippingMethodName]);
+                $shippingMethod = $shippingMethods = $this->getRepository('shipping_method')->findOneByName($shippingMethodName);
                 $channel->addShippingMethod($shippingMethod);
             }
         }
 
         if ($paymentMethodNames) {
             $paymentMethodNames = array_map('trim', explode(',', $paymentMethodNames));
-            $paymentMethods = $this->getRepository('payment_method')->findBy(['name' => $paymentMethodNames]);
-            foreach ($paymentMethods as $paymentMethod) {
+            foreach ($paymentMethodNames as $paymentMethodName) {
+                $paymentMethod = $this->getRepository('payment_method')->findOneByName($paymentMethodName);
```

Comment:
You know this is less optimal than previous logic? Anyway, let's keep it like this for now.